### PR TITLE
Mui card

### PR DESCRIPTION
**Pull request by TurtleWolfe**
added the heart icon for favorite

Screenshot 2022-02-17 at 19-47-33 🏍️ tMX

I'll work more on making it look like the prototype with stars and such tomorrow, but passing the props is scaffold in
Screenshot 2022-02-17 at 03-12-22 🏍️ tMX

### CODE REVIEW

File: src/components/muiCard.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// import * as React from 'react';
+// import svg_art from "../assets/coding_art.svg";
+
```
Comment (NullDefault) [inline src/components/muiCard.js:17]: filename should be different - all of your components are mui, why specify it.
In general, filenames should be the same as the Component name

File: src/components/muiCard.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import Card from '@mui/material/Card';
```
Comment (NullDefault) [inline src/components/muiCard.js]: Don't import these things one by one - you can import all the things in one line
i.e
import {Card, CardContent} from "@mui/material"
Look at the other files - it's done like that everywhere

File: src/components/muiCard.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { Button, CardActionArea, CardActions } from '@mui/material';
```
Comment (NullDefault) [inline src/components/muiCard.js]: Yeah everything should be imported like this

File: src/components/muiCard.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    >
+      <CardActionArea>
+        <CardMedia
+          component="img"
+          height="140"
+          // image="/static/images/cards/contemplative-reptile.jpg"
+          // alt="green iguana"
+          alt="funky illustration"
+          src={mapImage}
+        // src={svg_art}
```
Comment (NullDefault) [inline src/components/muiCard.js]: remove comments before eventual merge

**Review by NullDefault — CHANGES_REQUESTED** [review]

File: src/components/muiCard.js (hunk 1 above)
Comment (TurtleWolfe) [reply src/components/muiCard.js:17]: I was thinking to specify it as a custom component,
I don't mind changing the name
 but there are other places we may want to just use the standard Card and it gets confusing.

File: src/components/muiCard.js (hunk 2 above)
Comment (TurtleWolfe) [reply src/components/muiCard.js]: ok, I copied from the `docs`, but I see what you mean

File: src/components/muiCard.js (hunk 1 above)
Comment (NullDefault) [reply src/components/muiCard.js:17]: Can you elaborate on what you mean by the standard Card?

**Comment by TurtleWolfe** [comment]
I'm squashing this so Bobe can use it in his map Component

**Pull request merged**